azure-devops/run-build.yml

```diff
@@ -60,6 +60,14 @@ jobs:
         filename: 'azure-devops/validate-files.cmd'
         failOnStandardError: true
         arguments: '$(Build.ArtifactStagingDirectory)/tools/validate/validate.exe'
+    - task: PowerShell@2
+      displayName: 'Get Test Parallelism'
+      timeoutInMinutes: 2
+      inputs:
+        targetType: inline
+        script: |
+          $testParallelism = $env:NUMBER_OF_PROCESSORS - 2

```

Comment:
This is nice because it will also work if we change the VM size.

Reply:
No magic numbers 💯

Reply:
Though I guess 2 is a magic number 🤷‍♂